adafruit_platformdetect/chip.py

```diff
@@ -29,6 +37,32 @@ def id(self):
             return SAMD21
         if platform == "pyboard":
             return STM32
+        # Special case, if we have an environment var set, we could use FT232H
+        try:
```

Comment:
i think this code path won't get hit at all on linux systems, since they'll return from `self._linux_id()` above.

Reply:
oh yeah. fixed!